### PR TITLE
GEOPY-1855: Peak finder app crashes due to wild card

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1855 - Peak finder app crashes due to wild card**

### CODE REVIEW

**Comment by sebhmg** [comment]
haha the UI does not like it... ok, happy you caught it. 

In that case, also change in pylintrc, under `[DESIGN]`, value for `max-args`, and add entry `max-positional-arguments`
```
# crazy list or parameters for the UI elements
max-args=15
max-positional-arguments=15
```

**Comment by sebhmg** [comment]
or better than changing the global pylint config, disable the checks at the few problematic places with:
`# pylint: disable=too-many-positional-arguments, too-many-arguments`

```none
peak_finder/application.py:623:4: R0917: Too many positional arguments (14/5) (too-many-positional-arguments)
peak_finder/application.py:757:4: R0917: Too many positional arguments (11/5) (too-many-positional-arguments)
peak_finder/application.py:941:4: R0917: Too many positional arguments (17/5) (too-many-positional-arguments)
peak_finder/application.py:1038:4: R0917: Too many positional arguments (10/5) (too-many-positional-arguments)
peak_finder/application.py:1293:4: R0917: Too many positional arguments (10/5) (too-many-positional-arguments)
peak_finder/application.py:1437:4: R0917: Too many positional arguments (6/5) (too-many-positional-arguments)
peak_finder/application.py:1504:4: R0917: Too many positional arguments (9/5) (too-many-positional-arguments)
peak_finder/application.py:1740:4: R0917: Too many positional arguments (7/5) (too-many-positional-arguments)
peak_finder/application.py:1911:4: R0917: Too many positional arguments (15/5) (too-many-positional-arguments)
```

**Review by sebhmg — CHANGES_REQUESTED** [review]
ok, please just add `too-many-positional-arguments` to the pylint disable comments

**Comment by domfournier** [comment]
Somehow pylint doesn't know what `too-many-positional-arguments` is. Trying with the R0917 code instead

File: peak_finder/application.py [hunk 1]
```diff
@@ -36,6 +36,8 @@
 from peak_finder.params import PeakFinderParams
 from peak_finder.utils import get_ordered_survey_lines
 
+# pylint: disable=R0917
```
Comment (sebhmg) [inline peak_finder/application.py:39]:
```suggestion
# pylint: disable=too-many-positional-arguments
```

**Comment by sebhmg** [comment]
> Somehow pylint doesn't know what `too-many-positional-arguments` is. Trying with the R0917 code instead

it seemed it passed...

**Review by sebhmg — APPROVED** [review]

**Pull request merged**